test/__snapshots__/to-contain.spec.js.snap

```diff
@@ -1,10 +1,12 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`to-contain fails when the actual does not match the expected 1`] = `
+exports[`to-contain <ReactElement> fails when the actual does not match the expected 1`] = `
 "expected <Fixture /> to contain with all wrappers <User id={4} />
 
 the best match was
 <Fixture // should be <User
    // missing id={4}
 />"
 `;
+
```

Comment:
We need to investigate how we can make a more detailed inspection. But we can do that in a later PR.